src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
 import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Home from './components/Home';
+import Card from './components/Card';
+import Nav from './components/Nav';
+import { fetchData } from './Redux/data/dataSlice';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchData());
+  }, [dispatch]);
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
```

Comment:
- [optional] kindly add more unit tests for several components of the app and some pure functions in Redux code.
- the test you have some are failing. please fix them
![Screenshot (879)](https://user-images.githubusercontent.com/65298048/225156591-1281c23e-c120-4f20-b380-ba28a9a34767.png)

Reply:
Overnight the existing screenshot or remove the old one and run the test and it will pass.